src/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,148 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import type { ReactElement } from 'react';
+import { cloneElement } from 'react';
+import { merge } from '@utilities/merge';
+import { IconSize } from '@foundation/Icon';
+import { HelperText } from '@utilities/input';
+import { Validation } from '@utilities/validation';
+import { generateRandomId } from '@utilities/generateRandomId';
+import { Tooltip, TooltipProps } from '@components/Tooltip/Tooltip';
+
+export type LabelProps = {
+    text: string;
+    hugWidth?: boolean;
+    required?: boolean;
+    secondaryLabel?: string;
+    tooltips?: TooltipProps[];
+    toolTipSize?: IconSize;
+};
+
+export type FormFieldProps = {
+    children: ReactElement;
+    label: LabelProps;
+    disabled?: boolean;
+    error?: boolean;
+    errorText?: string;
+    errorTextColor?: Validation;
+    helperText?: string;
+    helperTextColor?: Validation;
+    hiddenLabel?: boolean;
+    readOnly?: boolean;
+    status?: Validation;
+    'data-test-id'?: string;
+};
+
+type FormatTooltipsProps = { tooltips: TooltipProps[] };
+
+const FormatTooltips = ({ tooltips }: FormatTooltipsProps): ReactElement[] => {
+    return tooltips.map((tip: TooltipProps) => (
```

Comment:
```suggestion
tooltips.map((tip: TooltipProps) => (
```
remove the return, you have only one child